core/controllers/collection_editor_test.py

```diff
@@ -101,20 +117,25 @@ def test_editable_collection_handler_get(self):
 
         # Check that non-editors cannot access the editor data handler.
         # This is due to them not being whitelisted.
-        self.get_json(
-            '%s/%s' % (
-                feconf.COLLECTION_EDITOR_DATA_URL_PREFIX,
-                self.COLLECTION_ID), expected_status_int=401)
+        url = '%s/%s' % (feconf.COLLECTION_EDITOR_DATA_URL_PREFIX, self.COLLECTION_ID)
+        
+        self.get_json(url, expected_status_int=401)
 
         # Check that whitelisted users can access the data
         # from the editable_collection_data_handler.
         self.login(self.EDITOR_EMAIL)
 
-        json_response = self.get_json(
-            '%s/%s' % (
-                feconf.COLLECTION_EDITOR_DATA_URL_PREFIX,
-                self.COLLECTION_ID))
+        json_response = self.get_json(url)
         self.assertEqual(self.COLLECTION_ID, json_response['collection']['id'])
+
+        # Check GET returns when an exceptio is raised in get learner
```

Comment:
typo - exception. And I think the comment should say something along the lines of "GET request errors with a 404 response when an exception is raised..."